ftw/tagging/profiles/default_plone5/skins.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<object name="portal_skins" allow_any="False" cookie_persistence="False">
+
+    <object name="ftw.tagging"
+            meta_type="Filesystem Directory View"
+            directory="ftw.tagging:skins/ftw_tagging" />
+
+    <skin-path name="*">
+        <layer name="ftw.tagging"
+               insert-after="custom" />
```

Comment:
Shame we can't lose skins ;-)

but `insert-after` is wrong - you mean `insert-before`..... which means `custom` can still override through the web (though we'd prob never do that in practise)